packages/medusa/src/api/routes/admin/batch/create-batch-job.ts

```diff
@@ -0,0 +1,51 @@
+import { IsBoolean, IsJSON, IsOptional, IsString } from "class-validator"
+import BatchJobService from "../../../../services/batch-job"
+import { validator } from "../../../../utils/validator"
+
+/**
+ * @oas [post] /batch
+ * operationId: "PostBatches"
```

Comment:
comment: See comment around consistency in operation ids
```suggestion
 * operationId: "PostBatchJobs"
```